sources/packages/backend/apps/api/src/route-controllers/student-scholastic-standings/models/student-scholastic-standings.dto.ts

```diff
@@ -35,3 +36,18 @@ export class ScholasticStandingSubmittedDetailsAPIOutDTO extends IntersectionTyp
   ScholasticStandingData,
   ActiveApplicationDataAPIOutDTO,
 ) {}
+
+/**
+ * Represents the possible errors that can happen during the
+ * application bulk withdrawal and provides a detailed description
+ * for every record that has an error.
+ */
+export interface ApplicationBulkWithdrawalValidationResultAPIOutDTO {
+  recordIndex: number;
+  sin?: string;
+  applicationNumber?: string;
+  withdrawalDate?: Date;
```

Comment:
correct me if I'm wrong, but this interface is used for record type 2 validation errors, right? so, even though there is a validation error, won't there be at least some values? In other words, do these fields need to be optional? 
![image](https://github.com/bcgov/SIMS/assets/77353155/88edc2ba-bdeb-44c3-97ce-19775fae5625)

Reply:
Just to mimic the empty values for the other records, we may be using them as optional

![Uploading image.png…]()

Reply:
@ann-aot looking at the validation result perspective, application number and withdrawal date may or may not be present in the validation response.